docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  camunda-bpm:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    environment:
+      - CAMUNDA_DB_PASSWORD=camunda
+      - CAMUNDA_DB_HOST=host.docker.internal
+      - CAMUNDA_DB_PORT=5434
+      - CLIENT_ID
+      - CLIENT_SECRET
+      - ISSUER_URI
+      - ENABLE_SCHEMA_UPDATE=true
+    ports:
+      - 8082:8080
```

I think what you had clashes with one of the reform apps :p
```suggestion
      - 8999:8080
```